src/cpu/cpu.cpp

```diff
@@ -2635,18 +2637,16 @@ void CPU_ShutDown([[maybe_unused]] Section* sec)
 #elif (C_DYNREC)
 	CPU_Core_Dynrec_Cache_Close();
 #endif
-	delete test;
```

Comment:
Why are we no longer destroying the class here?  I think you would reset the unique_pr here unless you want it get destroyed at some unspecified point when global objects get cleaned up.

Reply:
Good pickup, and you're right, I forgot about this. Calling `.reset()` on the unique_ptr on destroy.